prisma/schema.prisma

```diff
@@ -51,15 +51,34 @@ model User {
   telegram            String?     @db.VarChar
   github              String?     @db.VarChar
   hashed_ip_address   String?     @db.VarChar
-  events              Event[]
-  node_uptime         NodeUptime?
-  points              UserPoints?
+  redemption          Redemption?
+
+  events      Event[]
+  node_uptime NodeUptime?
+  points      UserPoints?
 
   @@index([email], name: "index_users_on_email")
   @@index([graffiti], name: "index_users_on_graffiti")
   @@map("users")
 }
 
+model Redemption {
+  id               Int          @id @default(autoincrement())
+  created_at       DateTime     @default(now()) @db.Timestamp(6)
+  updated_at       DateTime     @default(now()) @updatedAt @db.Timestamp(6)
+  user             User         @relation(fields: [user_id], references: [id])
+  user_id          Int          @unique
+  kyc_attempts     Int          @default(0)
+  kyc_status       KycStatus
+  jumio_account_id String?      @db.VarChar
+  public_address   String       @db.VarChar
+  email_verified   Boolean      @default(false)
+  dedupe_status    DedupeStatus
```

Comment:
I don't think this should live on the kyc rows. We can add eligibility to the user directly

Reply:
Yes, I think we should delete this its unrelated to this model.

Reply:
We decided to remove this